test/ConsoleAgent.js

```diff
@@ -27,13 +27,13 @@ describe('ConsoleAgent', function() {
 
   it('should translate objects', function(done) {
     frontendClient.once('Console.messageAdded', function(message) {
-      expect(message.message.parameters).to.deep.equal([{
-        type: 'object',
-        subtype: undefined,
-        objectId: 'console:1:1',
-        className: 'Object',
-        description: 'Object'
-      }]);
+      // expect(message.message.parameters).to.deep.equal([{
+      //   type: 'object',
+      //   subtype: undefined,
+      //   objectId: 'console:1:1',
+      //   className: 'Object',
+      //   description: 'Object'
+      // }]);
```

Comment:
I think the fact that you have to disable this test may indicate that this change is breaking console logs as I asked in https://github.com/node-inspector/node-inspector/pull/990/files#r107178707.